src/private/WRLDRouteView+Private.h

```diff
@@ -14,7 +14,21 @@
                           route:(WRLDRoute*)route
                          option:(WRLDRouteViewOptions*)options;
 
+-(void) addLinesForRoutePath:(CLLocationCoordinate2D*)path pathCount:(int) count ofColor:(UIColor*)color;
```

Comment:
Make params on multiple lines.

Reply:
Please mark as resolved if this and the rest of the comments are done.